api/v1alpha1/openapi.yaml

```diff
@@ -4550,9 +4495,9 @@ components:
             $ref: '#/components/schemas/ConfigProviderSpec'
         applications:
           type: array
-          description: List of applications.
+          description: List of application providers.
           items:
-            $ref: '#/components/schemas/ApplicationSpec'
+            $ref: '#/components/schemas/ApplicationProviderSpec'
```

Comment:
The previous type `ApplicationSpec` is currently being used in the UI.
I'll create a PR for updating the UI to the new types, so it's ready when this PR is merged.